cpp/src/io/parquet/stats_filter_helpers.hpp

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "io/parquet/parquet_common.hpp"
+
+#include <cudf/ast/detail/expression_transformer.hpp>
+#include <cudf/ast/expressions.hpp>
+#include <cudf/detail/utilities/vector_factories.hpp>
+#include <cudf/utilities/memory_resource.hpp>
+#include <cudf/utilities/span.hpp>
+#include <cudf/utilities/traits.hpp>
+
+namespace cudf::io::parquet::detail {
+
+/**
+ * @brief Base utilities for converting and casting stats values. Derived
+ * classes handle row group or page-level statistics as needed.
```

Comment:
```suggestion
 * @brief Base utilities for converting and casting stats values
 *
 * Derived classes handle row group or page-level statistics as needed.
```
The `@brief` is like a title and should not be multiple sentences.
(this file is not actually processed by doxygen but consistency is good practice in open source I think)